src/compiler/compilation.jl

```diff
@@ -3,12 +3,14 @@
 Base.@kwdef struct CUDACompilerParams <: AbstractCompilerParams
     cap::VersionNumber
     ptx::VersionNumber
+    link_libdevice::Bool # Used by Reactant.jl
```

Comment:
We're using `@kwdef`
```suggestion

    # for Reactant.jl, which raises NVVM intrinsics to MLIR
    link_libdevice::Bool = true
```